tests/MaxMind/Test/MinFraudTest.php

```diff
@@ -145,30 +142,32 @@ private function createNullRequest()
     }
 
     /**
-     * @expectedException \MaxMind\Exception\InvalidInputException
-     * @expectedExceptionMessage Must have keys
      * @dataProvider services
      *
      * @param mixed $class
      * @param mixed $service
      */
     public function testMissingIpAddress($class, $service)
     {
+        $this->expectException(InvalidInputException::class);
+        $this->expectExceptionMessage('Must have keys');
```

Comment:
Nice. I wasn't saying you had to do these ones as part of the story, but it is good that we won't have to deal with them later.

Reply:
Yeah, no worries. It was just a couple regexp finds/replaces.